<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>488</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>488</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MM32-LINK Configure</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>MM32_Logo.ico</normaloff>MM32_Logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>485</width>
      <height>254</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>500</height>
            </size>
           </property>
           <property name="title">
            <string>Firmware</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="2" column="0" colspan="2">
               <widget class="QPushButton" name="btnUpgrade">
                <property name="text">
                 <string>Upgrade</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QComboBox" name="cbbVersion">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>&amp;Device</string>
                </property>
                <property name="buddy">
                 <cstring>cbbDevice</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QComboBox" name="cbbDevice">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>&amp;Version</string>
                </property>
                <property name="buddy">
                 <cstring>cbbVersion</cstring>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="btnRefresh">
                <property name="text">
                 <string>Refresh</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="title">
            <string>Function</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="cboxBeep">
                <property name="text">
                 <string>Beep</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cboxPower">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Power</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbbPOut">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>3.3 V</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>5 V</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QPushButton" name="btnOK">
                <property name="text">
                 <string>Apply</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="btnClearLog">
                <property name="text">
                 <string>Clear log</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Message</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="1">
           <widget class="QTextBrowser" name="textView"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="processBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="format">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cbbDevice</tabstop>
  <tabstop>cboxPower</tabstop>
  <tabstop>cbbPOut</tabstop>
  <tabstop>cboxBeep</tabstop>
  <tabstop>btnClearLog</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClearLog</sender>
   <signal>clicked()</signal>
   <receiver>textView</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
